### DISCUSSION TITLE
Guidance on how to make a social editor with markdown and image upload support?

### DISCUSSION THREAD

**Discussion by bennidhamma** [remirror/remirror#293] [General]
Thanks for this great looking project. We're considering using it for a project. My apologies if an issue is the wrong medium, let me know if stackoverflow or something is better.

The social editor seems like a pretty good starting point for our needs. We use markdown for our comments, and also have a need to support image uploading.

Would a reasonable starting point be to fork the editor-social components and start splicing in the other bits? Or can the social editor be configured via props to a) support markdown and b) support image uploading?

If the former, would it make sense to hold off on this work until the next branch is further along? And if the latter, any guidance on how to do that would be greatly appeciated.

again, thank you for your work!

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/master/docs/contributing.md) document.

**Comment by ifiokjr** [comment]
@bennidhamma I've converted this to a discussion. 

My main focus, time permitting, has been on making the `next` version of the editor ready for production usage, hence the delay in the response.

Are you still considering remirror as a solution?
